### PR TITLE
Updates generic styles for Social Sharing on Dough and documentation …

### PR DESCRIPTION
**Pull request by davidtrussler**
…for the Made with Dough site

### CODE REVIEW

**Comment by slaywell** [comment]
:+1:

**Comment by tomas-stefano** [comment]
:+1:

File: assets/stylesheets/components/optional/_social_sharing_icons.scss [hunk 1]
```diff
@@ -1,11 +1,16 @@
 .social-sharing__item {
   display: inline-block;
   overflow: hidden;
+  margin: 0 10px 0 0;
 
   .social-sharing__item__icon {
```
Comment (nmiddleweek) [inline assets/stylesheets/components/optional/_social_sharing_icons.scss]: To follow BEM, this block should be moved outside so it isn't nested

File: assets/stylesheets/components/optional/_social_sharing_icons.scss [hunk 2]
```diff
@@ -1,11 +1,16 @@
 .social-sharing__item {
```
Comment (nmiddleweek) [inline assets/stylesheets/components/optional/_social_sharing_icons.scss]: The file name and classes should be the same.

So either _social-sharing.scss or change the top level class to .social_sharing_icons

**Pull request merged**